build/build_binary.sh

```diff
@@ -1,20 +1,15 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-DEBUG=${DEBUG:-""}
-if [ -n "$DEBUG" ]; then
-	set -x
-fi
-
 mkdir -p dist
 
 # populate tool versions
-BUILD_NUMBER=${BUILD_NUMBER:-"N/A"}
-CONTAINER_IMAGE_TAG=${CONTAINER_IMAGE_TAG:-"N/A"}
-NODE_VERSION=${NODE_VERSION:-"0"}
-YARN_VERSION=${YARN_VERSION:-"0"}
-WEBPACK_VERSION=${WEBPACK_VERSION:-"0"}
-GO_VERSION=${GO_VERSION:-"0"}
+BUILDNUMBER="N/A"
+CONTAINER_IMAGE_TAG="N/A"
+NODE_VERSION="0"
+YARN_VERSION="0"
+WEBPACK_VERSION="0"
+GO_VERSION="0"
```

Comment:
I assume this is to match EE, but doesn't this just make it so those variables cannot be overwritten? Is that what we want?

Reply:
It does, but they don't need to be overridden in this script since this script is only used for local development.
The CI pipeline uses the build_binary_azuredevops.sh instead which allows overrides.

Reply:
@hookenz I agree with @dakota-portainer there's nothing wrong with letting them be overridden (I think I put this change because I needed it once)